facebook-commerce.php

```diff
@@ -1465,6 +1496,37 @@ private function get_product_variation_attributes( array $variation ): array {
 		return $final_attributes;
 	}
 
+	/**
+	 * Update existing product using batch API.
+	 *
+	 * @param WC_Facebook_Product $woo_product
+	 * @param string              $fb_product_item_id
+	 * @return void
+	 */
+	public function update_product_item_batch_api( WC_Facebook_Product $woo_product, string $fb_product_item_id ): void {
+		$product = $woo_product->prepare_product( null, \WC_Facebook_Product::PRODUCT_PREP_TYPE_ITEMS_BATCH );
+		$requests = WC_Facebookcommerce_Utils::prepare_product_requests_items_batch( $product );
+
+		try {
+			$facebook_catalog_id = $this->get_product_catalog_id();
+			$response            = $this->facebook_for_woocommerce->get_api()->send_item_updates( $facebook_catalog_id, $requests );
+			if ( $response->handles ) {
```

Comment:
I noticed that in other places, we do this. We can keep this if we are sure about the `handles` property in the response:

``` php
if ( isset( $response_handles ) && is_array( $response_handles ) )